2023-05-13 00:43:51,003 - mysqlutil.py[line:38 ] - ERROR: 查询所有数据失败失败原因可能是：(1049, "Unknown database 'api_server'") 
